internal/provider/provider.go

```diff
@@ -21,6 +24,17 @@ func New() *schema.Provider {
 				Required:    true,
 				DefaultFunc: schema.EnvDefaultFunc("HYDRA_ADMIN_URL", nil),
 			},
+			"basic_auth_user": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("HYDRA_ADMIN_AUTH_USER", nil),
+			},
+			"basic_auth_pass": {
+				Type:        schema.TypeString,
+				Sensitive:   true,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("HYDRA_ADMIN_AUTH_PASS", nil),
```

Comment:
Let's specify auth method in the env variable as well to support other auth methods.

```suggestion
				DefaultFunc: schema.EnvDefaultFunc("HYDRA_ADMIN_BASIC_AUTH_PASSWORD", nil),
```